README.rst

```diff
@@ -26,6 +26,14 @@ we'll have to do this the long way.
 
     pip install -r requirements.txt
 
+#. Install the dependencies for Python 3 too::
+
+    pip3 install -r requirements.txt
+
+#. Install the Spacy English model::
+
+    python3 -m spacy download en
```

Comment:
instead, what about updating run_autonag.sh to have something like
```
if test ! -d spacy_files; then
python3 -m spacy download en
fi

```

Reply:
or not, it doesnt' really matter

Reply:
Soon, I'll fix bugbug to avoid this step and we'll remove the item from the README altogether.

Reply:
Done.